lib/new_relic/util/ssl.ex

```diff
@@ -0,0 +1,21 @@
+defmodule NewRelic.Util.SSL do
+  @moduledoc false
+
+  # See: https://github.com/hexpm/hex/blob/master/lib/hex/http/certs.ex
+  @cacertfile Path.expand("cacert.pem")
+  @external_resource @cacertfile
+  @cacerts File.read!(@cacertfile)
+           |> :public_key.pem_decode()
+           |> Enum.map(fn {:Certificate, der, _} -> der end)
+
+  def ssl_options() do
+    [
+      ssl: [
+        verify: :verify_peer,
+        cacerts: @cacerts,
```

Comment:
I would put cacert.pem in the package's /priv directory and pass `cacertfile` instead. Passing certificates/keys as DER binaries instead of paths is not tested very well, and there have been issues with it in past OTP releases. The caching by the ssl application works quite well, so there is no negative impact.

Reply:
That's a good idea, I'll do that. I got this technique from the code in Hex but this does seem safer.